kotlin/build.gradle

```
@@ -1,25 +1,47 @@
 plugins {
     id 'java'
-    id 'org.jetbrains.kotlin.jvm' version '1.3.11'
+    id 'org.jetbrains.kotlin.jvm' version '1.5.31'
+    id 'application'
+}
+
+ext {
+    kotlinVersion = '1.5.31'
 }
 
 group 'com.gu'
 version '1.0-SNAPSHOT'
 
-sourceCompatibility = 1.8
-
 repositories {
     mavenCentral()
 }
 
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
-    testCompile group: 'junit', name: 'junit', version: '4.12'
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8"
+    testImplementation group: 'junit', name: 'junit', version: '4.12'
+    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:1.5.31"
```

We can just use the kotlin version constant here:
```suggestion
    testImplementation "org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion"
```